oap-server/server-storage-plugin/storage-influxdb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/influxdb/query/EBPFProfilingTaskQuery.java

```diff
@@ -122,7 +122,7 @@
             task.setProcessName((String) values.get(6));
             task.setTaskStartTime((long) values.get(7));
             task.setTriggerType(EBPFProfilingTriggerType.valueOf((int) values.get(8)));
-            task.setFixedTriggerDuration((long) values.get(9));
+            task.setFixedTriggerDuration(((Number) values.get(9)).longValue());
             task.setTargetType(EBPFProfilingTargetType.valueOf((int) values.get(10)));
             task.setCreateTime((long) values.get(11));
             task.setLastUpdateTime((long) values.get(12));
```

Comment:
I think these types should be unreliable. need to change like Line 125
pls recheck